src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -430,149 +453,125 @@ private LoadDocumentEntity LoadDocument(PostedDataEntity postedData, bool loadAl
                 }
                 loadDocumentEntity.SetPages(pageData);
             }
+
             loadDocumentEntity.SetGuid(documentGuid);
             loadDocumentEntity.SetShowGridLines(globalConfiguration.Viewer.GetShowGridLines());
+            
             // return document description
             return loadDocumentEntity;
         }
 
-
-        private dynamic GetHandler()
-        {
-            if (viewerHtmlHandler != null)
-            {
-                return viewerHtmlHandler;
-            }
-            else
-            {
-                return viewerImageHandler;
-            }
-        }
-
-        private PageDescriptionEntity GetPageDescriptionEntities(PageData page)
+        private PageDescriptionEntity GetPageDescriptionEntities(Page page)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetPageDescriptionEntities' a static method.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5242614)